.travis.yml

```diff
@@ -1,7 +1,50 @@
-sudo: false
-language: javascript
-
-install: npm install
-
-
-script: npm run test:unit:js && npm run test:smoke:typescript
+jobs:
+  include:
+    - stage: test
+      sudo: false
+      language: javascript
+      install: npm install
+      script: npm run test:unit:js && npm run test:smoke:typescript
+    - stage: build android
+      language: android
+      sudo: false
+      android:
+        components:
+         - tools
+         - platform-tools
+         - tools
+         - build-tools-26.0.2
+         - android-26
+         - extra-google-m2repository
+         - extra-android-m2repository
+      env:
+        - ANDROID_TARGET=android-26 ANDROID_ABI=armeabi-v7a
+        - ADB-INSTALL_TIMEOUT=5
+      before_install:
+        - yes | sdkmanager "platforms;android-26"
+        - yes | sdkmanager "platforms;android-23"
+      install:
+        - curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+        - sudo apt-get install -y nodejs
+      script:
+        - cd examples/plain
+        - npm install
+        - ./node_modules/.bin/react-native link
+        - cd android
+        - ./gradlew assembleRelease
+      licenses:
+        - 
+    - stage: build cocoa
+      language: objective-c
+      osx-image: xcode7.3
+      env: SDK=macosx10.11 BUILD_OSX=1
```

Comment:
Is it possible to use a more modern version of XCode? Most people will be using 9 these days